src/create_methods.js

```diff
@@ -32,14 +32,19 @@ export default (context, config) => {
         )
         .validator({ clean: true }),
         run({ _id, ...doc }) {
-          // console.log('updating', collectionName, _id, doc);
-          if (!isAllowed(collectionName, this.userId)) {
-            throw new Meteor.Error('not allowed', 'You are not allowed');
-          }
+          //console.log('updating', collectionName, _id, doc);
+          if (Meteor.isServer) {
+            if (!isAllowed(collectionName, this.userId)) {
+              throw new Meteor.Error('not allowed', 'You are not allowed');
+            }
 
-          const updated = collection.update(_id, { $set: doc });
-          if (updated === 0) {
-            throw new Meteor.Error('not found', 'Entry not found');
+            // need bypassCollection2: true it could not update the whoe doc anymore
+            const updated = collection.update(
+              _id, { $set: doc }, { bypassCollection2: true },
+            );
+            if (updated === 0) {
+              throw new Meteor.Error('not found', 'Entry not found');
+            }
```

Comment:
this needs an explanation. Is there an issue with simple-schema that has been tracked?